src/index.js

```diff
@@ -536,14 +713,25 @@ function _getUserAccount(engine, payload, cb) {
 			id = payload.id;
 		}
 		if(cb) {
```

Comment:
can we write this function in async await manner. callbacks often create confusion. we can do it later it would be great to add a todo here.

Reply:
This function actually returns a promise if callback function is not passed as an argument. See line 726. So you can use await while calling this function without callback.